app.js

```diff
@@ -1,11 +1,96 @@
 /* Imports */
-
+import { renderVillain } from './utils.js';
 /* Get DOM Elements */
-
+const villainsEl = document.getElementById('villains');
+const playerHPEl = document.getElementById('player-hp');
+const playerImgEl = document.getElementById('player-img');
+const inputEl = document.getElementById('villain-input');
+const buttonEl = document.getElementById('villain-button');
+const defeatedCountEl = document.getElementById('defeated-count');
 /* State */
+let defeatedCount = 0;
+let playerHp = 10;
 
+const villains = [
+    {
+        name: 'Bowser',
+        hp: 5,
+    },
+    {
+        name: 'Goomba',
+        hp: 3,
+    },
+    {
+        name: 'Boo',
+        hp: 2,
+    },
+];
 /* Events */
+buttonEl.addEventListener('click', () => {
+    const villainName = inputEl.value;
+
+    if (!villainName) {
+        return;
+    }
+
+    const newVillain = {
+        name: villainName,
+        hp: Math.ceil(Math.random() * 5),
+    };
+
+    villains.push(newVillain);
+
+    inputEl.value = '';
 
+    displayVillains();
+});
 /* Display Functions */
 
-// (don't forget to call any display functions you want to run on page load!)
+function displayVillains() {
+    villainsEl.textContent = '';
+
+    for (let villain of villains) {
+        const newVillainEl = renderVillain(villain, playerHp);
+        // newVillainEl.setAttribute('id', 'game-over-button');
+
+        newVillainEl.addEventListener('click', () => {
+            if (playerHp <= 0) {
+                alert('Mama Mia!');
+                return;
+            }
+            if (Math.random() > 0.5) {
+                alert('Lets a go! Mario hit' + ' ' + villain.name);
+                villain.hp--;
+
+                if (villain.hp === 0) {
+                    defeatedCount++;
```

Comment:
i think you need to throw a disable into here somehow, because you need to be able to disable the default goblins as well as the newly rendered ones.